proto/ibc/applications/transfer/v1/genesis.proto

```diff
@@ -16,4 +17,8 @@ message GenesisState {
     (gogoproto.moretags)     = "yaml:\"denom_traces\""
   ];
   Params params = 3 [(gogoproto.nullable) = false];
+  // total_escrowed contains the total amount of tokens escrowed
+  // by the transfer module
+  repeated cosmos.base.v1beta1.Coin total_escrowed = 4
+      [(gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins", (gogoproto.nullable) = false];
```

Comment:
does this need yaml? Seems like v7 is still using yaml tags looking at the above